nextcord/__init__.py

```diff
@@ -10,10 +10,10 @@
 
 """
 
-__title__ = 'discord'
+__title__ = 'nextcord'
 __author__ = 'tag-epic & Rapptz'
 __license__ = 'MIT'
-__copyright__ = 'Copyright 2015-present Rapptz'
+__copyright__ = 'Copyright 2015-present Rapptz & tag-epic'
```

Comment:
similar license header as referenced above
```suggestion
__copyright__ = 'Copyright 2015-2021 Rapptz & 2021-present tag-epic'
```